src/web/client/test/integration/schemaHelperUtil.test.ts

```diff
@@ -0,0 +1,791 @@
+        //Act
+        const result = {
+            value: [
+                {
+                    adx_portallanguageid_value: "1",
+                    adx_websitelanguageid: "english",
+                },
+                {
+                    adx_portallanguageid_value: "2",
+                    adx_websitelanguageid: "hindi",
+                },
+                { adx_websitelanguageid: "french" },
+            ],
+        };
+
+        const expectedResult = new Map<string | null, string>([
+            ["english", "1"],
+            ["hindi", "2"],
+            ["french", Constants.PORTAL_LANGUAGE_DEFAULT],
+        ]);
+        const schema = "portalschemav1";
+        //Action
+        const resultData = getwebsiteLanguageIdToPortalLanguageMap(
+            result,
+            schema
+        );
+        //Assert
+        expect(resultData).deep.equals(expectedResult);
+    });
+
+    it("getwebsiteLanguageIdToPortalLanguageMap_shouldReturnEmptyWhenValueEmptyArray", () => {
+        //Act
+        const result = { value: null };
+
+        const schema = "portalschemav1";
+        //Action
+        const resultData = getwebsiteLanguageIdToPortalLanguageMap(
+            result,
+            schema
+        );
+        //Assert
+        expect(resultData).empty;
+    });
+
+    it("getwebsiteLanguageIdToPortalLanguageMap_shouldReturnEmptyWhenValueUndefined", () => {
+        //Act
+        const result = { value: undefined };
+
+        const schema = "portalschemav1";
+        //Action
+        const resultData = getwebsiteLanguageIdToPortalLanguageMap(
+            result,
+            schema
+        );
+        //Assert
+        expect(resultData).empty;
+    });
+
+    it("getwebsiteLanguageIdToPortalLanguageMap_shouldReturnEmptyWhenValueHavingEmptyArray", () => {
+        //Act
+        const result = { value: [] };
+
+        const schema = "portalschemav1";
+        //Action
+        const resultData = getwebsiteLanguageIdToPortalLanguageMap(
+            result,
+            schema
+        );
+        //Assert
+        expect(resultData).empty;
+    });
+
+    it("getwebsiteLanguageIdToPortalLanguageMap_shouldReturnEmptyWhenResultIsNull", () => {
+        //Act
+        const result = null;
+
+        const schema = "portalschemav1";
+        //Action
+        const resultData = getwebsiteLanguageIdToPortalLanguageMap(
+            result,
+            schema
+        );
+        //Assert
+        expect(resultData).empty;
+    });
+
+    it("getwebsiteLanguageIdToPortalLanguageMap_shouldReturnEmptyWhenResultIsUndefined", () => {
+        //Act
+        const result = undefined;
+
+        const schema = "portalschemav1";
+        //Action
+        const resultData = getwebsiteLanguageIdToPortalLanguageMap(
+            result,
+            schema
+        );
+        //Assert
+        expect(resultData).empty;
+    });
```

Comment:
useOctetStreamContentType and isWebFileV2 these method?

Reply:
Done , "isWebFileV2" is already written